whelk-core/src/main/resources/ext/marcframe.json

```diff
@@ -8968,21 +8969,33 @@
         "marcDefault": "0"
       },
       "$f": {"property": "date"},
-      "$g": {"addProperty": "comment"},
-      "$h": {"property": "marc:mediaTerm"},
-      "$x": {"addLink": "identifiedBy", "resourceType": "ISSN", "property": "value"},
-      "spec_": [
+      "$g": {"property": "qualifier"},
+      "$h": {"ignored": true, "NOTE:LC": "Medium - ignore"},
```

Comment:
Doesn't the global changes need to take care of these changes as well?
- marc:previousTitle.Title.comment (list) --> hasTitle.FormerTitle.qualifier (no list)
- Remove all marc:previousTitle.Title.marc:mediaTerm
- Remove all marc:isAddedEntry with value...?

Reply:
Ah see your note in the description now about "Manual fix 6 instances of comment and marc:mediaTerm > qualifier" :)

Reply:
@sarwik nope, too few to bother, manual fix, see checklist :)